Automattic/delphin
# Add `RELATED_WORD_FETCH_FAIL` action to `state.ui.domainSearch.relatedWords` reducer

This PR adds a case to the related words reducer to handle if the API request to Wordnik fails. I didn't add any code to indicate that this case is any different than the case where the Wordnik request succeeds but returns no related words because:
- I don't think it is important to message this to the user. `No related words for [keyword] were found.` seems sufficient for both cases.
- This is an edge case, and probably not worth special attention other than ensuring that the app doesn't break or show a perpetual loading state.

**Note:** The first four commits are from #577 and enable `Search`. They will be removed before merging.

**Testing**
- Add `127.0.0.1 api.wordnik.com` to your hosts file.
- Visit `/search`.
- Enter a keyword and hit space.
- Click on the keyword.
- Assert that `No related words for [keyword] were found.` is displayed instead of `Loading…`.
